sklearnex/dispatcher.py

```diff
@@ -289,6 +292,14 @@ def patch_sklearn(name=None, verbose=True, global_patch=False, preview=False):
             "for scikit-learn >= 0.22 only ..."
         )
 
+    if not no_msg and "sklearn" in sys.modules.keys():
```

Comment:
i would say would worth printing out submodules list that we see already imported from scikit?

Reply:
I did it locally and it's always a pretty long list, not so sure if that's useful.

```
$ python -c "import json,sys;from sklearn.linear_model import LogisticRegression;print(json.dumps(list(sys.modules.keys()), indent=2, sort_keys=True))" | grep sklearn                                         
  "sklearn._config",                                             
  "sklearn.__check_build._check_build",
  "sklearn.__check_build",                              
  "sklearn._distributor_init",                                          
  "sklearn.exceptions", 
  "sklearn.utils._joblib", 
  "sklearn.utils._bunch",                                  
  "sklearn.utils._metadata_requests",                                
  "sklearn.utils.metadata_routing",
  "sklearn.utils._estimator_html_repr",                             
  "sklearn.externals",                                        
  "sklearn.externals._packaging",                 
  "sklearn.externals._packaging._structures",
  "sklearn.externals._packaging.version",
  "sklearn.utils.deprecation",            
  "sklearn.utils.fixes",    
  "sklearn.utils._array_api",       
  "sklearn.utils._isfinite",
  "sklearn.utils.validation",              
  "sklearn.utils._param_validation",
  "sklearn.utils.class_weight",
  "sklearn.utils.discovery",        
  "sklearn.utils.murmurhash",                    
  "sklearn.utils",                   
  "sklearn.utils._available_if",    
  "sklearn.utils._set_output",  
  "sklearn.utils._tags",    
  "sklearn.base",   
  "sklearn.utils._openmp_helpers", 
  "sklearn.utils._show_versions",       
  "sklearn",                      
  "sklearn.utils._logistic_sigmoid",
  "sklearn.utils.sparsefuncs_fast", 
  "sklearn.utils.extmath",  
  "sklearn.utils.sparsefuncs",    
  "sklearn.utils._encode",                   
  "sklearn.utils._mask",
  "sklearn.preprocessing._encoders",
  "sklearn.preprocessing._data",
  "sklearn.utils.stats",
  "sklearn.preprocessing._discretization",
  "sklearn.utils.metaestimators",     
  "sklearn.preprocessing._function_transformer",
  "sklearn.utils.multiclass",     
  "sklearn.preprocessing._label",
  "sklearn.preprocessing._csr_polynomial_expansion",
  "sklearn.preprocessing._polynomial",
  "sklearn.preprocessing._target_encoder_fast",
  "sklearn.preprocessing._target_encoder",
  "sklearn.preprocessing", 
  "sklearn.utils._random",
  "sklearn.utils._seq_dataset",
  "sklearn.utils.parallel",
  "sklearn.linear_model._base",
  "sklearn.linear_model._bayes",
  "sklearn.utils._response",     
  "sklearn.utils._plotting",       
  "sklearn.metrics.cluster._bicluster",
  "sklearn.metrics.cluster._expected_mutual_info_fast",
  "sklearn.metrics.cluster._supervised",
  "sklearn.metrics._dist_metrics",
  "sklearn.metrics._pairwise_distances_reduction._datasets_pair",
  "sklearn.utils._cython_blas",              
  "sklearn.metrics._pairwise_distances_reduction._base",
  "sklearn.metrics._pairwise_distances_reduction._middle_term_computer",
  "sklearn.utils._heap",         
  "sklearn.utils._sorting",     
  "sklearn.metrics._pairwise_distances_reduction._argkmin",
  "sklearn.metrics._pairwise_distances_reduction._argkmin_classmode",
  "sklearn.utils._vector_sentinel",
  "sklearn.metrics._pairwise_distances_reduction._radius_neighbors",                                                                                                                                          
  "sklearn.metrics._pairwise_distances_reduction._dispatcher",
"sklearn.metrics._pairwise_distances_reduction",
  "sklearn.metrics._pairwise_fast",
  "sklearn.metrics.pairwise",
  "sklearn.metrics.cluster._unsupervised",
  "sklearn.metrics.cluster",
  "sklearn.metrics._classification",
  "sklearn.metrics._plot",
  "sklearn.metrics._plot.confusion_matrix",
  "sklearn.metrics._base",
  "sklearn.metrics._ranking",
  "sklearn.metrics._plot.det_curve",
  "sklearn.metrics._plot.precision_recall_curve",
  "sklearn.metrics._plot.regression",
  "sklearn.metrics._plot.roc_curve",
  "sklearn.metrics._regression",
  "sklearn.metrics._scorer",
  "sklearn.metrics",
  "sklearn.model_selection._split",
  "sklearn.model_selection._validation",
  "sklearn.model_selection._plot",
  "sklearn.utils.random",
  "sklearn.model_selection._search",
  "sklearn.model_selection",
  "sklearn.linear_model._cd_fast",
  "sklearn.linear_model._coordinate_descent",
  "sklearn._loss._loss",
  "sklearn._loss.link",
  "sklearn._loss.loss",
  "sklearn._loss",
  "sklearn.utils.optimize",
  "sklearn.linear_model._linear_loss",
  "sklearn.linear_model._glm._newton_solver",
  "sklearn.linear_model._glm.glm",
  "sklearn.linear_model._glm",
  "sklearn.linear_model._huber",
  "sklearn.utils.arrayfuncs",
  "sklearn.linear_model._least_angle",
  "sklearn.svm._bounds",
  "sklearn.svm._liblinear",
  "sklearn.svm._libsvm",
  "sklearn.svm._libsvm_sparse",
  "sklearn.svm._base",
  "sklearn.svm._classes",
  "sklearn.svm",
  "sklearn.utils._weight_vector",
  "sklearn.linear_model._sgd_fast",
  "sklearn.linear_model._sag_fast",
  "sklearn.linear_model._sag",
  "sklearn.linear_model._logistic",
  "sklearn.linear_model._omp",
  "sklearn.linear_model._stochastic_gradient",
  "sklearn.linear_model._passive_aggressive",
  "sklearn.linear_model._perceptron",
  "sklearn.linear_model._quantile",
  "sklearn.linear_model._ransac",
  "sklearn.linear_model._ridge",
  "sklearn.linear_model._theil_sen",
  "sklearn.linear_model"
```